infoschema/infoschema.go

```diff
@@ -347,6 +353,31 @@ func HasAutoIncrementColumn(tbInfo *model.TableInfo) (bool, string) {
 	return false, ""
 }
 
+// PolicyByName is used to find the policy.
+func (is *infoSchema) PolicyByName(name string) (*placement.Policy, bool) {
+	is.policyMutex.RLock()
+	defer is.policyMutex.RUnlock()
+	t, r := is.policyMap[name]
+	return t, r
+}
+
+// SetPolicy is used to set the policy.
+func (is *infoSchema) SetPolicy(policy *placement.Policy) {
```

Comment:
I think it can be removed like `DeletePolicy`. If builder is using the method, we can make it private or move it into builder

Reply:
sure